site/content/blog/eth2-dev-chain.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Setting up private Ethereum 2.0 development chain
+date: 2020-03-26T09:00:00-00:00
+image: /images/blog/ethereum20-validator.png
+draft: false
+keywords:
+- ethereum 2.0
+- proof of staking
+- dev
+- lighthouse
+- chainguardian
+description: Learn how to setup eth2 development chain using lighthouse client. 
+---
+
+> Disclaimer: This article is highly technical intended for developers and eth2 enthusiasts.
+
+### Why would I setup private eth2 chain?
+
+If you had any experience with development on eth1, you probably already know answer to this question so 
```

Comment:
In this case this might not be needed as it is really short paragraph and good to read anyway. Can be a bit on the offensive side too.

You can start with something like: "In case you have experience with development on eth1, you may be familiar with the problem of fast testing and using private networks as a solution." That way you say immediately what's this section about and people can choose to skip or not.